src/global/shared-types/component.types.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Allowed component
+ */
+export interface CustomComponent {
```

Comment:
I think we need another name for this. Even though no type named `CustomCompoent` is being exported from this package today, the term is used in several other places and has another meaning so it might be confusing. Also, you can in theory allow regular HTML elements here as well, so maybe the term "element" is better than "component". `AllowedElement` perhaps?

Reply:
I was hoping to reuse this interface in the text editor part when we add custom nodes. Can we call it `CustomElement` ?

Reply:
I we make it private for now you can call it whatever you want until we know more about what it will be used for :slightly_smiling_face:

Reply:
see https://github.com/Lundalogik/lime-elements/pull/3157/commits/f72651ea26500dae156fad30e1a4cc6f7a877214